integration-cli/docker_cli_run_test.go

```diff
@@ -2718,6 +2718,23 @@ func TestRunAllowPortRangeThroughExpose(t *testing.T) {
 	logDone("run - allow port range through --expose flag")
 }
 
+// test docker run expose a invalid port
+func TestRunExposePort(t *testing.T) {
+	runCmd := exec.Command(dockerBinary, "run", "--expose", "80000", "busybox")
+	out, _, err := runCommandWithOutput(runCmd)
+	//expose a invalid port should with a error out
+	if err != nil && !strings.Contains(out, "Invalid range format for --expose") {
```

Comment:
@cpuguy83  You are right, since the invalid port will always with a error out, `if err == nil` is a good way. Thank you for the tips